config/locales/en.yml

```diff
@@ -486,3 +486,9 @@ en:
     transparency: See all our transparency data
     written_statement: See all our written statements to Parliament
     authored_article: See all our authored articles
+  broken_links:
```

Comment:
Could we get the english translations added in the previous commit, then we can review it together? Regenerating the remaining translations is fine for this commit.